_posts/2024-03-20-strimzi-kraft-migration.md

```diff
@@ -0,0 +1,265 @@
+---
+layout: post
+title:  "Migrate your Strimzi-operated cluster from ZooKeeper to KRaft"
+date: 2024-03-20
+author: paolo_patierno
+---
+
+In the previous [blog post](https://strimzi.io/blog/2024/03/21/kraft-migration/) we explored how Apache ZooKeeper is used to store Apache Kafka clusters metadata and what are the current limitations.
+We introduced the new KRaft protocol which allows to overcome such limitations and store metadata in Kafka itself, taking into account that the ZooKeeper support will be removed soon.
```

Comment:
`We` -> `The Apache Kafka project` or ``The Apache Kafka community`

Reply:
As "We" I was referring to the previous blog post where the authors gave an introduction on the KRAft protocol. But I can see it could be misleading, I will change with your suggestion.